tests/test_onnx_export.py

```diff
@@ -80,12 +80,17 @@ def to_numpy(tensor):
     return tensor.cpu().numpy()
 
 
-def set_layer_scale(module: torch.nn.Module, scale: float):
-    module.fp8_init()
-    module.fp8_meta["scaling_fwd"].scale = torch.ones(
-        2, dtype=torch.float32, device="cuda") / scale
-    module.fp8_meta["scaling_fwd"].scale_inv = torch.ones(
-        2, dtype=torch.float32, device="cuda") * scale
+def set_layer_scale(module: torch.nn.Module, scales: float, num_gemms: int=1):
+    module.fp8_init(num_gemms=num_gemms)
+    assert len(scales) == num_gemms * 2, "Each gemm should be accompanied by 2 scales"
```

Comment:
I don't think we need this assert because it checks a TE predicate that should not affect the export process.